db_functions.py

```diff
@@ -952,8 +1035,10 @@ def handle_matched_filesets(
 
 def delete_original_fileset(fileset_id, conn):
     with conn.cursor() as cursor:
+        print(fileset_id)
```

Comment:
I think this is a leftover. You may consider introducing like 'debug mode' and print this information